hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -445,7 +444,7 @@ public void testCreateTableCustomSortOrder() throws Exception {
 
   @Test
   public void testDatabaseAndNamespaceWithLocation() throws Exception {
-    Namespace namespace1 = Namespace.of("noLocation");
+    Namespace namespace1 = Namespace.of("nolocation");
```

Comment:
For some reason, in Hive 4, the `org.apache.hadoop.hive.metastore.api.Database` below (`database1`) returns all-lowercase "nolocation.db" when `getLocationUri()` is called.

Reply:
I need to check, however, as I recall Hive lower-cases all identifiers: db, table, partition

Reply:
difference is probably because Hive doesn't normally use `HiveCatalog#createNamespace` only `HMS RestCatalog`
````
database.setLocationUri(new Path(getExternalWarehouseLocation(), namespace.level(0)).toString() + ".db");
````